homeassistant/components/poolstation/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to your Poolstation account",
+        "description": "[%key:common::config_flow::description::confirm_setup%]",
```

Comment:
:point_up: This description does not fit here, it is used if no data has to be entered.